src/components/appcontent.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC, useState } from 'react';
+import DashboardIcon from '../assets/dashboard.svg';
+import DataUsageIcon from '../assets/data_usage.svg';
+import DCOpenOrdersIcon from '../assets/dc_open_orders.svg';
+import LeaderboardIcon from '../assets/leaderboard.svg';
+import LegendToggleIcon from '../assets/legend_toggle.svg';
+import Navbar from './common/navbar';
+import SearchField from './common/searchfield';
+import SidePaneGrid from './common/sidepanegrid';
+import SidePaneList from './common/sidepanelist';
+
+const MENU_LIST = [
```

Comment:
Keep this list in a utils file so that it is easy to configure in the future

Reply:
ok